src/app/core/database/database.service.provider.ts

```diff
@@ -26,8 +26,5 @@ import { SessionService } from "../session/session-service/session.service";
  */
 export let databaseServiceProvider = {
```

Comment:
I would inline this `{ provide: Database, useClass: PouchDatabase }` in the module directly, as it is very straightforward now.